File: 3768-check-if-digits-are-equal-in-string-after-operations-i/check-if-digits-are-equal-in-string-after-operations-i.js
/**
 * @param {string} s
 * @return {boolean}
 */
var hasSameDigits = function(s) {
    do {
        let tem = ""
        for(let i=1;i<s.length;i++){
            tem+=String((Number(s[i-1])+Number(s[i]))%10)
        }
        s = tem
    } while(s.length!=2)
    return s[0]===s[1]
};

File: 3768-check-if-digits-are-equal-in-string-after-operations-i/check-if-digits-are-equal-in-string-after-operations-i.py
class Solution:
    def hasSameDigits(self, s: str) -> bool:
        while len(s)!=2:
            tem = ""
            for i in range (1, len(s)):
                tem+=str((int(s[i-1])+int(s[i]))%10)
            s = tem
        return s[0]==s[1]
        